pkg/components/velero/component.go

```diff
@@ -201,12 +201,8 @@ func (c *component) getProvider() (provider, error) {
 		providers = append(providers, c.Restic)
 	}
 
-	if len(providers) > 1 {
-		return nil, fmt.Errorf("more than one provider configured")
-	}
-
-	if len(providers) == 0 {
-		return nil, fmt.Errorf("no providers configured")
+	if len(providers) != 1 {
+		return nil, fmt.Errorf("cannot have more than one provider configured, got: %d", len(providers))
```

Comment:
`"cannot have more than one provider configured, got: 0"` doesn't make sense.

Reply:
@invidian I have removed one conditional statement and revised the error message. If you think having two conditionals make sense for a better error message then I have no problems.